pkg/internal/checkup/checkup.go

```diff
@@ -456,6 +456,7 @@ func (c *Checkup) waitForPVCBound(ctx context.Context, result, errStr *string) {
 
 	log.Printf("Waiting for PVC %q bound", pvcName)
 	if err := wait.PollImmediateWithContext(ctx, pollInterval, time.Minute, conditionFn); err != nil {
+		log.Printf("PVC %q failed to bound", pvcName)
```

Comment:
Do we explain in anyway why the PVC bound failed?

Reply:
Usually in this case the PVC is stuck in Pending phase without any status Conditions updated, so it simply timeouts waiting for Bound. It usually means the provisioner does not support such small PVC.

Reply:
yes sure, though there can be other reasons why PVC failed to bound but I see it is difficult to know this in the test. thanks